test/php/gutenberg-coding-standards/Gutenberg/Sniffs/CodeAnalysis/ForbiddenFunctionsAndClassesSniff.php

```diff
@@ -3,8 +3,7 @@
  * Gutenberg Coding Standards.
  *
  * @package gutenberg/gutenberg-coding-standards
- * @link    https://github.com/WordPress/gutenberg
- * @license https://opensource.org/licenses/MIT MIT
```

Comment:
It is not clear to me why we were including the license in each sniff and now we are not anymore. I'm not sure if this is necessary or not. What is the reasoning behind this change?

Reply:
Well spotted.
Fixed in cff54f744d890d095ec5079616d27afb61fdd02a.

Reply:
The reasoning behind this change is to align the organization of licensing information with that of Gutenberg PHP files. Since Gutenberg PHP files do not include licensing information and instead specify the license type in the composer.json file, the GBCS package, integral to the Gutenberg project, adopt the same approach. This ensures consistency and adherence to Gutenberg practices, thereby avoiding confusion.

Reply:
Makes sense to me, thanks for the additional details. For future PRs, I suggest including this information in the commit message.